plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1724,9 +1788,20 @@ def onUpdateScribbles(self):
             # get scribbles + label
             segmentationNode = self._segmentNode
             labelmapVolumeNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLLabelMapVolumeNode")
-            slicer.modules.segmentations.logic().ExportVisibleSegmentsToLabelmapNode(
-                segmentationNode, labelmapVolumeNode, self._volumeNode
+            save_segment_ids = vtk.vtkStringArray()
+            segmentationNode.GetSegmentation().GetSegmentIDs(save_segment_ids)
```

Comment:
is the "background" label useful for the scribbles / crf algorithm? If not, I might suggest removing that segment ID here

Reply:
Current binary graphcut method may not need it. 
Background will be required for multi-label CRF-based methods (see my last message on slack regarding multi-label). 
I will therefore keep this and handle any label selection on server side...